reference/catalog/product-variants_catalog.v3.yml

```diff
@@ -1272,7 +1272,12 @@ paths:
                               x-nullable: true
                             inventory_level:
                               type: integer
-                              description: 'Inventory level for the variant, which is used when the product’s inventory_tracking is set to `variant`.'
+                              description: |- 
+                                Current inventory level of the product. Simple inventory tracking must be enabled (See the `inventory_tracking` field) for this to take any effect.
```

Comment:
actually you don’t need tracking mode to be enabled. UI hides form but API endpoints(both catalog and inventory) allows to set stock for untracked products

Reply:
@alex-gaiduchok Thanks, I accidentally changed the sentence, so I just changed it back to how it was before